docker-compose.yml

```diff
@@ -38,6 +38,16 @@ services:
       - rabbitmq
       - redis
 
+  celery-beat:
```

Comment:
when I try to use docker-compose I have an error:


`ERROR: for open-notificaties_celery-beat_1  Cannot start service celery-beat: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "/celery_beat.sh": stat /celery_beat.sh: no such file or directory: unknown
`